cmake/Modules/GrMiscUtils.cmake

```diff
@@ -72,6 +70,7 @@ endmacro(GR_ADD_CXX_COMPILER_FLAG_IF_AVAILABLE)
 function(GR_LIBRARY_FOO target)
     #set additional target properties
     set_target_properties(${target} PROPERTIES SOVERSION ${LIBVER})
+    set_target_properties(${target} PROPERTIES VERSION ${VERSION})
```

Comment:
This will actually set versioning of our shared libraries like so:
```
╰─ ls -a ~/.usrlocal/lib/libgnuradio-analog.so*                                      ✔  15:43:04
/home/andrej/.usrlocal/lib/libgnuradio-analog.so
/home/andrej/.usrlocal/lib/libgnuradio-analog.so.3.8git
/home/andrej/.usrlocal/lib/libgnuradio-analog.so.3.8tech-preview-365-gd66bc948
```

```
lrwxrwxrwx 1 andrej users      27 May 16 16:15 /home/andrej/.usrlocal/lib/libgnuradio-analog.so -> libgnuradio-analog.so.3.8.0
lrwxrwxrwx 1 andrej users      29 May 16 16:15 /home/andrej/.usrlocal/lib/libgnuradio-analog.so.3.8.0 -> libgnuradio-analog.so.3.8.0.0
```

Reply:
FYI, in UHD, we set both these to the ABI version. I have no clue if that's better or worse, but it's what Maitland suggested.